client/components/gsuite/docs/new-user-list.tsx

```diff
@@ -59,6 +59,7 @@ const GSuiteNewUserListExample = (): React.FunctionComponent => {
 				onUsersChange={ ( changedUsers ) => setUsers( changedUsers ) }
 				users={ users }
 				onReturnKeyPress={ () => void 0 }
+				showLabels={ true }
```

Comment:
Can we remove the flag rather than switching the value of the prop?

Our long-term goal has been to show labels in all cases, so making that be the only option is going to be simpler for us in the long run.